### PR TITLE
Add mkdocs documentation

### PR DESCRIPTION
**Pull request by jmaupetit**
After discussing with @willdurand, we conclude that sphinx + readthedoc wasn't a nice/easy combo and decided to give mkdocs a try. Since I love markdown and ReStructuredText is ... ReStructuredText, I think this PR will replace #73

### CODE REVIEW

**Comment by k4nar** [comment]
It looks very good :+1: .

Maybe the "Getting started" section could show a little bit more about the functionalities, like `watson log` or `watson report`. It's in the readme and in the front page, so this is the section which is the more likely to be read.

**Comment by jmaupetit** [comment]
Done.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Contributing
+
+If you are reading this, we thank you in advance for willing to contribute to the Watson project! You are awesome.
+
+Before starting to contribute, please read the following:
+
+* [How to contribute](https://tailordev.github.io/Watson/contributing/how-to/)
+* [How to start hacking](https://tailordev.github.io/Watson/contributing/hack/)
+* [Pull request guidelines](https://tailordev.github.io/Watson/contributing/pr-guidelines/)
+* [Contributor Code of Conduct](https://tailordev.github.io/Watson/contributing/coc/)
```
Comment (k4nar) [inline CONTRIBUTING.md]: Do we have to hardcode `tailordev.github.io` ? This means that we cannot see the doc offline.

File: README.rst [hunk 2]
```diff
@@ -3,469 +3,66 @@ Watson
...
-Tell Watson when you start working on a task with the ``start`` command.
-Then, when you are done with this task, stop the timer with the ``stop``
-command. This will create what we call a **frame**. That's pretty much
-everything you need to know to start using Watson.
+Wanna know what it looks like? Check this below.
 
-Each frame consists of the name of a project and some tags. Your tags
-can be shared across projects and can be used to generate detailed
-reports.
+![screenshot](docs/img/screenshot.png)
```
Comment (k4nar) [inline README.rst]: The image doesn't show up, but maybe it's just because it's not merged yet :/ .

Comment (jmaupetit) [inline README.rst]: Yes it is.

File: CONTRIBUTING.md (hunk 1 above)
Comment (jmaupetit) [inline CONTRIBUTING.md]: I have now idea on how to support both off- and on-line mode. This could be achieved later no?

Comment (willdurand) [inline CONTRIBUTING.md]: use relative URLs?

Comment (k4nar) [inline CONTRIBUTING.md]: yup, I guess that would work.

Comment (jmaupetit) [inline CONTRIBUTING.md]: Ok so we don't point to the github pages anymore.

Comment (jmaupetit) [inline CONTRIBUTING.md]: I am sorry I might be missing the point, but after digging around, I don't see to what relative URLs should point to? Urls from github pages (`contributing/hack/`) are different from repository files relative paths (`docs/contributing/hack.md`).

Comment (k4nar) [inline CONTRIBUTING.md]: Well, can I build the doc locally to have my own instance ?

Comment (jmaupetit) [inline CONTRIBUTING.md]: Yes, but users browsing our repository and alerted about CONTRIBUTING instructions won't.

Comment (k4nar) [inline CONTRIBUTING.md]: Oh yes I know that. I was just saying that in case someone want to embed our doc somewhere.

Comment (jmaupetit) [inline CONTRIBUTING.md]: So can we postpone this?

Comment (k4nar) [inline CONTRIBUTING.md]: Yes, it's not a blocker.

**Comment by k4nar** [comment]
Very nice docs! Great job.

I think everything is ready, we'll this "offline docs" point later.

File: README.rst (hunk 2 above)
Comment (jmaupetit) [inline README.rst]: Finally it wasn't :)
https://github.com/TailorDev/Watson/commit/d168b2b79f3bc521fd4ed7198cb26e0a4e42e973

**Comment by willdurand** [comment]
@jmaupetit do we have auto-publish with Travis-CI?

**Comment by jmaupetit** [comment]
Do you mean like #78?

**Pull request merged**